README.md

```diff
@@ -1 +1,19 @@
 # hw01
+
+##Background Information:
+My name is **_Natalia Joseph_**
+![Photo of me](https://images.app.goo.gl/iYvQDpmX9QDD6icQ6)
+I am currently enrolled in the *Committee on International Relations* program.
+I recently moved to Chicago from Mohyliv-Podilskyi, Ukraine, where I taught English. If you want more information about Mohyliv-Podilskyi, I suggest you visit the [link to the Mohyliv-Podilskyi Wikipedia page!](https://en.wikipedia.org/wiki/Mohyliv-Podilskyi)
```

Comment:
This link was embedded perfectly, and you formatted it into the biography really well!